### PR TITLE
Update cosmosac

### PR DESCRIPTION
**Pull request by pw0908**
Added functionalities where, if COSMO-SAC parameters are not in the Clapeyron database, they will be looked up in the NIST/COSMOSAC database.

@ianhbell do you have any opposition to this? We can add the relevant references to the model to ensure both your database and the folks who built the database receive the appropriate recognition.

### CODE REVIEW

**Comment by ianhbell** [comment]
That makes sense to me. I would recommend a strong comment in the docs, perhaps a print statement in the console that you must manually silence, indicating the license terms within which the COSMO-SAC files are made available. They are not allowed to be used for all purposes.

**Comment by pw0908** [comment]
Great! I've changed the implementation so that users must intentionally select the nist database to use it, at which point a warning is thrown to let them know to check the license.

The docs themselves are getting an overhaul on a separate branch where I've added similar warnings.

File: src/models/Activity/COSMOSAC/utils.jl
Comment (longemen3000) [inline src/models/Activity/COSMOSAC/utils.jl:1]: We could have this file as `COSMOSAC_Identifiers.csv`,already downloaded. In the long term, we should have the inchikey in the Identifiers.csv file. Anyway, seems good to merge

**Review by longemen3000 — APPROVED** [review]

File: src/models/Activity/COSMOSAC/utils.jl
Comment (pw0908) [reply src/models/Activity/COSMOSAC/utils.jl:1]: I've added this.

**Comment by longemen3000** [comment]
test fail is unrelated

**Pull request merged**